client/landing/stepper/index.tsx

```diff
@@ -63,49 +56,37 @@ function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	reduxStore.dispatch( requestSites() );
 }
 
-interface configurableFlows {
-	flowName: string;
-	pathToFlow: Flow;
+function determineFlow() {
+	const flowNameFromPathName = window.location.pathname.split( '/' )[ 2 ];
+
+	let flow = availableFlows.find( ( f ) => f.flowName === flowNameFromPathName ) || {
+		flowName: 'site-setup-flow',
+		downloadFlow: () => import( './declarative-flow/site-setup-flow' ),
+	};
+
+	if ( isAnchorFmFlow() ) {
+		flow = {
+			flowName: 'anchor-fm-flow',
+			downloadFlow: () => import( './declarative-flow/anchor-fm-flow' ),
+		};
```

Comment:
I'd prefer to put these 2 flows to `availableFlows` so that people won't feel confused about where these 2 come from.

Besides, I assume we have to give them chunk names too